### ISSUE TITLE
Reorder data in coordinate maps for better memory accesses

### ISSUE THREAD

**Issue by circlingthesun** [circlingthesun/test#43]
ala the quadrant packing described in the original paper, except that it probably makes more sense to store each quadrant in contiguous memory rather than using the 2x2 layout from the paper.

**Comment by circlingthesun** [comment]
[bmerry] This should probably be deferred until after #10.

**Comment by circlingthesun** [comment]
[bmerry] I tried this, and it make no difference to performance since these cache misses are not the bottleneck.
